build.gradle

```diff
@@ -169,15 +169,32 @@ configure(opentelemetryProjects) {
         mavenLocal()
     }
 
-    sourceCompatibility = 1.8
-    targetCompatibility = 1.8
```

Comment:
@anuraaga I think by removing this, Intellij doesn't configure the language level correctly.